qadence/backends/gpsr.py

```diff
@@ -110,7 +114,9 @@ def multi_gap_psr(
     batch_size = max(t.size(0) for t in param_dict.values())
 
     # get shift values
-    shifts = shift_prefac * torch.linspace(PI / 2 - PI / 5, PI / 2 + PI / 5, n_eqs)
+    shifts = shift_prefac * torch.linspace(
+        PI / 2 - PI / 4, PI / 2 + PI / 5, n_eqs
```

Comment:
Why does it become PI / 4 instead of PI / 5 @vytautas-a ?

Reply:
Short answer - it allows to pass all the differentiation tests. Long answer - I suspect that due to some symmetries in matrices computed during GPSR calculations, results with some generators are not correct. Breaking this symmetry of shifts around `pi/2` helps in this regard.